File: Domain/Requestor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.Contracts;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Requestor
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }

        [EmailAddress]
        public string Email { get; set; }
        public int HidtaId { get; set; }
        public Hidta Hidta { get; set; }
        public ICollection<TaskItem> TaskItems { get; set; }
    }
}

File: client-app/src/models/Requestor.ts
export interface IRequestor {
    id: number;
    firstName: string;
    lastName: string;
    email?: string
  }
  
  export class Requestor implements IRequestor {
    id: number;
    firstName: string;
    lastName: string;
    email?: string
  
    constructor(init: { id: number; firstName: string, lastName: string, email: string }) {
      this.id = init.id;
      this.firstName = init.firstName;
      this.lastName = init.lastName;
      this.email = init.email;
    }
  }
  
  // export class TaskItemFormValues {
  //     id: number = 0;
  //     description: string = '';
  //     date: Date | null = null;
  
  //     constructor(taskItem?: TaskItemFormValues) {
  //         if (taskItem) {
  //             this.id = taskItem.id;
  //             this.description = taskItem.description;
  //             this.description = taskItem.description;
  //         }
  //     }
  // }
  